2025/build-tooling/slides.md

```diff
@@ -16,61 +16,59 @@ is: feedback
 
 ---
 
-# Audience
+# Are you looking to...
 
-- Developers looking to add build step to their apps
-- Developers looking to learn more about the capabilities of the build tools
-- Developers interested in how Esri's teams are leveraging modern build tools
+- Add a build step to your app?
+- Learn more about build tool capabilities?
+- Learn how Esri teams are leveraging modern build tools?
+
+Then this presentation is for you!
 
 
 ---
 
 # Agenda
 
-TODO: create agenda based on the presentation outline
-
-
+- Overview
+- Demos
+  - Get started with Vite
+  - Add dependencies
+  - Use TypeScript and modern features
+  - Lazy load source bundles
+  - Publish the app
+  - Add tests with Vitest
+  - Add custom plugins
 
 ---
 
-# Benefits of using build tools
+# What are build tools?
 
-- Ease of using external libraries
-- Can import/export between files
-- Live reload
-- Better performance - minification and bundle splitting
-- Can use modern JS syntax, which will get downcompiled as needed
-- Can build complete offline apps with local assets
-- TypeScript, IntelliSense
-  - TODO: Can borrow material from my previous year's TypeScript Benefits list https://github.com/maxpatiiuk/esri-dev-summit-presentations/blob/main/2024/calcite-react/index.md#typescript-
-  - show example of bugs TypeScript can easily catch?
-  - or show autocomplete for Maps SDK APIs?
-- more customizability, control, support for advanced use cases
-
-TODO: split into multiple slides or reduce. Move TypeScript out further down?
+Build tools automate tasks like bundling and optimizing source code, help manage dependencies, and enhance development workflows. Examples: Vite, Parcel, Webpack...
 
+(Thanks Copilot)
 
 ---
 
-# Disadvantages of build tools
-
-- More complexity to set them up and get going
-  - But, build tools are getting easier and easier to use - the barrier have never been lower
+# Build tool benefits
 
+- Improve development experience with live reload and hot module replacement
+- Enable modern JavaScript syntax features and dependencies
+- Automate bundling, minifying, and transpiling source code
+- Optimize performance by splitting bundles and reducing file sizes
+- Add plugins for additional capabilities, like tests
```

Comment:
```suggestion
- Improve development experience (live reload and Hot Module Replacement)
- Enable modern JavaScript syntax and dependencies
- Automate transpiling, bundling and testing
- Optimize performance by splitting bundles and reducing file sizes
- Allow extending capabilities with plugins
```

Comments:
- transpiling happens before minifying
- instead of "transpiling" could also say downpiling or transforming, which are more self-explanatory terms
- you have "minifying"  twice (minifying on line 3, and reducing file sizes on line 4
- build tools add support for plugins, rather than directly add plugins

Reply:
Makes sense, updated and also made a few more minor changes, like "live updates" to match below